### PR TITLE
Slightly clarify what users should specifically do with the .android/.ios folders

### PR DESCRIPTION
**Pull request by xster**
fixes https://github.com/flutter/flutter/issues/21706

### CODE REVIEW

File: src/docs/development/add-to-app/android/project-setup.md [hunk 1]
```diff
@@ -94,14 +94,20 @@ $ cd some/path/
...
+folder contains an Android project that can both help you run a barebones
+standalone version of your Flutter module via `flutter run` and it's also a
+wrapper that helps bootstrap the Flutter module an embeddable Android library.
 
 {{site.alert.note}}
-Add custom Android code to your existing application or a plugin, not to
-the module in `.android/`. Changes made in `.android/` aren't embedded in your existing application.
-Regenerate the directory by running `flutter clean` or `flutter pub get` in the
-`my_flutter` directory.
+Add custom Android code to your own existing application'sproject  or a plugin,
```
Comment (jmagman) [inline src/docs/development/add-to-app/android/project-setup.md]: Missing space "application'sproject"

File: src/docs/development/add-to-app/ios/project-setup.md [hunk 2]
```diff
@@ -39,30 +39,34 @@ application:
...
-├─.ios/
-│ ├─Runner.xcworkspace
-│ └─Flutter/podhelper.rb
-├─lib/
-│ └─main.dart
-├─test/
-└─pubspec.yaml
+├── .ios/
+│   ├── Runner.xcworkspace
+│   └─ Flutter/podhelper.rb
```
Comment (jmagman) [inline src/docs/development/add-to-app/ios/project-setup.md]: Is is missing a `─` if you're adding one to each.

File: src/docs/development/add-to-app/android/project-setup.md (hunk 1 above)
Comment (xster) [reply src/docs/development/add-to-app/android/project-setup.md]: 🤦🏻‍♂️ thanks

File: src/docs/development/add-to-app/ios/project-setup.md (hunk 2 above)
Comment (xster) [reply src/docs/development/add-to-app/ios/project-setup.md]: 🤦🏻‍♂️

**Pull request merged**